src/notification.rs

```diff
@@ -242,8 +241,8 @@ impl<'a> Notification<'a> {
                 _ => "no",
             }),
             NSString::from_str(match self.sound {
-                Some(sound) if check_sound(sound) => sound,
```

Comment:
Is the `check_sound` not needed anymore?

Reply:
Yeah it's hard to determine if a sound is valid, the OS pretty much checks the entire file system :joy:

Reply:
what is the verdict here? leave it in?